opm/grid/common/WellConnections.cpp

```diff
@@ -398,24 +397,33 @@ computeDefunctWellNames(const std::vector<std::vector<int> >& wells_on_proc,
         }
     }
 
-    // Compute defunct wells in parallel run.
-    std::vector<int> defunct_wells(wells.size(), true);
+    // Compute wells active/inactive in parallel run.
+    // boolean indicates whether the well perforates local cells
+    std::vector<std::pair<std::string,bool>> parallel_wells;
```

Comment:
This type is used in many places, perhaps a type alias (`WellsPresent` for example) would be good?